toolip/utils/config.py

```diff
@@ -0,0 +1,33 @@
+from enum import Enum
+
+from pydantic import BaseSettings
+
+
+class EnvironmentEnum(str, Enum):
+    production = 'production'
+    development = 'development'
+    test = 'test'  # for pytest
+    qa = 'qa'
+
+
+class MainConfiguration(BaseSettings):
+    environment: EnvironmentEnum = 'development'  # type: ignore
+    public_domain: str = 'mydomain.satel.ca'
+    sentry_environment: str = 'unknown'
```

Comment:
@hillairet I added this property because it existed in all the repos listed in the spreadsheet **except for sb-pim** who doesn't have sentry related properties on the config class. Thoughts on this?

Reply:
What are the typical values for the sentry environment in the apps where it's used?

Reply:
For the default, it's the same for all of them which is `'unknown'` and an empty string for the `sentry_dsn` property. Pretty much copy-pasted I would say.

Reply:
@qw-in Could use your guidance here :point_up:

Reply:
In development sentry should be disabled. In deployed environments it is generally:
"production", "staging" kinda thing

Reply:
So how about `sentry_environment: Optional[SentryEnvironment] = None` with `SentryEnvironment` an Enum with just "production" and "staging"?
Obviously the rest of the code, which probably should be in Toolip too, should handle `None` as meaning that sentry should be off.